File: javascript/src/models/stripeConfig.ts
// this file is @generated
/* eslint @typescript-eslint/no-explicit-any: 0 */

export interface StripeConfig {
  secret: string;
}

export const StripeConfigSerializer = {
  _fromJsonObject(object: any): StripeConfig {
    return {
      secret: object["secret"],
    };
  },

  _toJsonObject(self: StripeConfig): any {
    return {
      secret: self.secret,
    };
  },
};

File: rust/src/models/stripe_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct StripeConfig {
    pub secret: String,
}

impl StripeConfig {
    pub fn new(secret: String) -> Self {
        Self { secret }
    }
}
